src/server.js

```diff
@@ -64,6 +67,12 @@ module.exports = function() {
 		return res.sendFile(theme);
 	});
 
+	app.get("/packages/:package/:style.css", (req, res) => {
```

Comment:
I don't really see a reason to hardcode this for `.css`, just let plugins specify which files they want to serve.

Reply:
Hmmm alright, that makes it a little more complicated, but I'll look into it.

Reply:
Alright, changing this to be for any type of file was quite easy, I've got that working. However, combining it with the 1 above (along with keeping the backwards compatibility of the themes) is quite difficult.

Reply:
What's the trouble?

If you mean it's difficult for default themes because they don't have `package.json`, I wanted to eventually create them.

Reply:
Alright, well I've combined this with the themes if we can assume that all of the old ways of defining themes was just getting it from "themes/something.css" if we can assume that, I can keep the backwards compatibility. I'll upload this evening hopefully.

Reply:
Actually no scratch all of that, I missed something. Combining the 2 is quite hard when we have to support the local themes as well. For just now, I'll keep it separate. Until we move the themes into packages.

Reply:
Will this work for packaged themes?

Reply:
Yeah, I can combine them for packaged themes easily, but not for the local ones.

Reply:
> What's the trouble?

Reply:
> What's the trouble?

Oh sorry, I missed this before. It's a bit complicated to explain, but here we go. 

It's a combination of the default theme config, allowing the old way of defining things, and combining the `/themes/theme.css` and `/packages/package/file` endpoints. It obviously makes sense to remove `/themes/theme.css` and just have `/packages/package/file` as the themes are just files.

When you are doing this, you have to have a package name, which we would either do as the name of the theme (possibility) or something like `localTheme` or something horrible. Not so difficult just now, I guess. You just define the local theme path as `/packages/localTheme/default.css` and then we switch on the package, if it's called `localTheme` then we go get the theme file from the themes folder.

So, the default theme can be either `name` or `path/to.css`. Now, if it's a name, we have to know whether or not it's a local theme, but that's easy because we have a list of themes and can get the appropriate filepath for the theme (we built up a `localTheme` object for each local theme at the start). However, the complication comes when we have a path.

It's simple if we can define them all as `themes/default.css` or similar, because then we can pull the name out of there. But we can't guarantee that, as we support any path to any css file (currently). So we could have something like `themes/folder/test.css`, or even something like `otherfolderhere/something.css`. 

The best way to do this is probably to remove the support for specifying a css file, only support local files in the themes folder, and then we can combine the 2.

Does that make sense? Have I missed something obvious?

Reply:
And then on top of that, deprecating local themes, so we can remove them in the future and only allow the default theme built into the lounge. We could even automatically install the other themes, and as we will have removed path support for css files, then the name would map to the package rather than the local theme.

So I would suggested:

1. Ship this PR in 2.7.0 with the 2 backwards compatible ways of dealing with things, but deprecated support for local themes (I can add that into the PR before it gets merged, just letting the user know if they have a local theme that isn't 1 of the themes we ship with and telling them to get it packaged)
2. Package up the 4 themes (we could just package up the 3 non-default themes, but that would still leave some hardcoded stuff for the default theme)
3. Have the lounge automatically install the default theme and the 3 non-default themes (we could just ship with the 1 theme, but it would be nice to the users if we didn't)
4. In 3.0 Remove the deprecated support for css file paths
5. After 3.0, we can then remove the support for local themes (if we were automatically installing them, then it wouldn't be a breaking change as long as the css file paths support has already been removed)

After all those things have been done, we can very easily combine the 2, because all the themes are packaged up, and we don't have to do any weird stuff.

Thoughts?